Use built-in alertmanager config if no fallback config is provided (#3541)
grafana/mimir: docs/sources/operators-guide/architecture/components/alertmanager.md

#### Fallback configuration

When a tenant doesn't have a Alertmanager configuration, the Grafana Mimir Alertmanager uses a fallback configuration.
By default, there is always a fallback configuration set.

> The default fallback configuration can be overwritten using the `-alertmanager.configs.fallback` command-line flag.

Reviewer: ```suggestion
You can overwrite the default fallback configuration via the `-alertmanager.configs.fallback` command-line flag.
```

Reviewer: Rationale: passive to active voice
